typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow/decorators/helpers/sourceFile.ts

```diff
@@ -12,19 +12,20 @@
  * This product may include a number of subcomponents with separate copyright notices and license terms. Your use of these subcomponents is subject to the terms and conditions of the subcomponent's license, as noted in the LICENSE file.
  * #L%
  */
-import { MethodDeclaration, SourceFile, SyntaxKind, factory } from "typescript";
+import { MethodDeclaration, SourceFile, SyntaxKind, addSyntheticLeadingComment, factory } from "typescript";
 import { printSourceFile } from "../../../../helpers/source";
 import { createWorkflowItemPrologueStatements } from "../../../../codeTransformers/prologueStatements";
+import { WorkflowItemDescriptor } from "../../../../../decorators";
 
 export interface SourceFilePrinter {
-	printSourceFile(methodNode: MethodDeclaration, sourceFile: SourceFile): string;
+	printSourceFile(methodNode: MethodDeclaration, sourceFile: SourceFile, itemInfo?: WorkflowItemDescriptor): string;
 }
 
 /**
  * Default source file printer will directly print the source file with the method node body statements.
  */
 export class DefaultSourceFilePrinter implements SourceFilePrinter {
-	public printSourceFile(methodNode: MethodDeclaration, sourceFile: SourceFile): string {
+	public printSourceFile(methodNode: MethodDeclaration, sourceFile: SourceFile, itemInfo: WorkflowItemDescriptor): string {
```

Comment:
The `itemInfo` parameter should be optional (as in the interface is optional.

Reply:
In the interface it should not be optional, will fix it